presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

```diff
@@ -299,7 +300,8 @@ class TaskManagerTest : public testing::Test {
   void writeToFile(
       const std::string& filePath,
       const std::vector<RowVectorPtr>& vectors) {
-    auto options = std::make_shared<dwio::common::WriterOptions>();
+    std::shared_ptr<dwio::common::WriterOptions> options =
```

Comment:
@kewang1024 @xiaoxmeng Does the following not work?

const std::shared_ptr<dwio::common::WriterFactory> writerFactory_;

In constructor, initialize the writerFactory_ as:
writerFactory_(dwio::common::getWriterFactory(dwio::common::FileFormat::DWRF))

Then in writeToFile:

auto options = writerFactory_->createWriterOptions();

This way we don't need to include "velox/dwio/dwrf/writer/Writer.h" but just the dwio::common ones.

Reply:
Yeah, I think it works as in the production code. Thanks!

Reply:
Thanks @yingsu00 for your good suggestion!